testsuite/forge.py

```diff
@@ -463,11 +474,14 @@ def now(self) -> datetime:
 
 
 class FakeTime(Time):
-    _now: datetime = datetime.fromisoformat("2022-07-29T00:00:00+00:00")
+    _now: datetime = datetime.fromtimestamp(1659078000, timezone.utc)
 
     def now(self) -> datetime:
         return self._now
 
+    def epoch(self) -> str:
+        return "1659078000"
```

Comment:
Make this a constant, since it's used twice? Or just return `_now` as a unix timestamp string?